osmaxx-py/excerptconverter/converter_manager.py

```diff
@@ -0,0 +1,39 @@
+from excerptconverter.baseexcerptconverter import BaseExcerptConverter
+
+
+class ConverterManager:
+    @staticmethod
+    def converter_configuration():
+        export_options = {}
+        for Converter in BaseExcerptConverter.available_converters:
+            export_options[Converter.__name__] = Converter.converter_configuration()
+        return export_options
```

Comment:
This function can be rewritten as a single [dictionary comprehension](https://www.python.org/dev/peps/pep-0274/):

    @staticmethod
    def converter_configuration():
        return {Converter.__name__: Converter.converter_configuration()
                for Converter in BaseExcerptConverter.available_converters}